src/format/KeePass2.cpp

```diff
@@ -133,3 +129,29 @@ KeePass2::ProtectedStreamAlgo KeePass2::idToProtectedStreamAlgo(quint32 id)
         return KeePass2::ProtectedStreamAlgo::InvalidProtectedStreamAlgo;
     }
 }
+
+QString KeePass2::cipherToString(QUuid cipherUuid)
```

Comment:
Does it make sense to cover these 2 new methods by unit tests?
BTW, I am a fan of unit testing and ready to help with this on the project. I saw 64% of total coverage, it can be defiantly improved.

Reply:
I agree, sometimes very trivial logic can be tested indirectly.